protos/sources/vega/events/v1/events.proto

```diff
@@ -567,6 +567,15 @@ message ProtocolUpgradeStarted {
 }
 
 
+// CoreSnapshotData represents the core snapshot data.
+message CoreSnapshotData {
+  // The block height at which snapshot was taken
+  uint64 block_height = 1;
+
+  // The hash of the snapshot block
+  string block_hash = 2;
```

Comment:
Could we sent the core version along with the event? So that when listing from the datanode you can see which snapshots can be used with the current version of core?

Reply:
we can, good idea.